plugins/source/aws/resources/services/ec2/key_pairs.go

```diff
@@ -77,3 +89,8 @@ func fetchEc2KeyPairs(ctx context.Context, meta schema.ClientMeta, parent *schem
 	res <- output.KeyPairs
 	return nil
 }
+func resolveEc2keyPairArn(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	return client.ResolveARNWithRegion(client.EC2Service, func(resource *schema.Resource) ([]string, error) {
```

Comment:
Not sure if there's a better way to do this at the moment while keeping `gen.hcl` and the code from drifting